docs/02_cleos/03_command-reference/push/push-transaction.md

```diff
@@ -1,36 +1,163 @@
 
-`-j,--json` - print result as json
+### Options
+- `-h,--help` - Print this help message and exit
+- `-x,--expiration` - Set the time in seconds before a transaction expires, defaults to 30s
+- - `-f,--force-unique` - Force the transaction to be unique. this will consume extra bandwidth and remove any protections against accidently issuing the same transaction multiple times
+`-s,--skip-sign` - Specify if unlocked wallet keys should be used to sign transaction
+- `-j,--json` - Print result as JSON
+- `--json-file` _TEXT_ - Save result in JSON format into a file
+- `-d,--dont-broadcast` - Don't broadcast transaction to the network (just print to stdout)
+- `--return-packed` - Used in conjunction with --dont-broadcast to get the packed transaction
+- `-r,--ref-block` _TEXT_ - Set the reference block num or block id used for TAPOS (Transaction as Proof-of-Stake)
+- `--use-old-rpc` - Use old RPC push_transaction, rather than new RPC send_transaction
+- `--compression` _ENUM:value in {none->0,zlib->1} OR {0,1}_ - Compression for transaction 'none' or 'zlib'
+- `-p,--permission` _TEXT_ - An account and permission level to authorize, as in 'account@permission'
+- `--max-cpu-usage-ms` _UINT_ - Set an upper limit on the milliseconds of cpu usage budget, for the execution of the transaction (defaults to 0 which means no limit)
+- `--max-net-usage` _UINT_ - Set an upper limit on the net usage budget, in bytes, for the transaction (defaults to 0 which means no limit)
+- `--delay-sec` _UINT_ - Set the delay_sec seconds, defaults to 0s
+- `--sign-with` _TEXT_ - The public key or json array of public keys to sign with
+- `-o,--read-only` - Specify a transaction is read-only
+- `-t,--return-failure-trace` - Return partial traces on failed transactions, use it along with --read-only)
 
-`-d,--dont-broadcast` - don't broadcast transaction to the network (just print to stdout)
 
-`-p,--permission` _Type: Text_ - An account and permission level to authorize, as in 'account@permission'
+## Requirements
+* Install version 2.2 or greater of `cleos.`
+[[info | Note]]
+| The cleos tool is bundled with the EOSIO software. [Installing EOSIO](../../00_install/index.md) will also install the cleos comand line tool and the keosd key store or wallet. 
 
-`--max-cpu-usage-ms` _UINT_ - set an upper limit on the milliseconds of cpu usage budget, for the execution of the transaction (defaults to 0 which means no limit)
+## Examples
 
-`--max-net-usage` _UINT_ - set an upper limit on the net usage budget, in bytes, for the transaction (defaults to 0 which means no limit)
+1. Basic usage
+```shell
+cleos push transaction {}
+```
 
-`--delay-sec` _UINT_ - set the delay_sec seconds, defaults to 0s
+2. Call a read-only query
+```shell
+cleos push transaction --read-only -j '{"actions":[{"account":"scholder","name":"get","authorization":[{"actor":"bob","permission":"active"}],"data":""}]}' -p bob@active
```

Comment:
Target account and permission again don't match, which is not usual for simple examples.

Reply:
I've updated the where explanation to make this clearer - but again don't see why you would use the same account as the smart contract.